packages/modules/http/api.py

```diff
@@ -18,3 +19,22 @@ def create_request_function(url: str, path: Optional[str]) -> Callable[[], float
         return lambda: None
     else:
         return functools.partial(_request_value, url + path)
+
+
+def fetch_soc(soc_url: str, range_url: str, vehicle: int) -> Union[int, float]:
+    if soc_url is None or soc_url == "none":
+        soclog.warn("http_soc: soc_url not defined - set soc to 0")
+        soc = 0
+    else:
+        soc_text = req.get_http_session().get(soc_url, timeout=5).text
+        soclog.debug("http_soc: soc_text="+soc_text)
+        soc = int(soc_text)
+    if range_url is None or range_url == "none":
+        soclog.warn("http_soc: range_url not defined - set range to 0.0")
+        range = float(0)
+    else:
+        range_text = req.get_http_session().get(range_url, timeout=5).text
+        soclog.debug("http_soc: range_text="+range_text)
+        range = float(range_text)
+    soclog.info("http_soc: soc="+str(soc)+", range="+str(range))
```

Comment:
```suggestion
```
Die abgefragten Werte werden automatisch beim Publishen an den Broker geloggt: [https://github.com/openWB/core/blob/0880c9e56e46760e5c753873d13521b31fc2e027/packages/modules/common/store/_api.py#L26](url) Dann müssen die Logmeldungen nicht in jedem Modul enthalten sein.

Reply:
OK, ich hatte die Meldungen auch gesehen, aber die im Modul nach dem testen erst mal stehen lassen.
Sind jetzt raus wie vorgeschlagen.